### DISCUSSION TITLE
[WEB]  'dart:js_interop' is not available on this platform.

### DISCUSSION THREAD

**Discussion by Nicola-Cestaro** [flutter-stripe/flutter_stripe#1825] [Q&A]
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/flutter_stripe_web-6.0.0-dev.0/lib/src/widgets/card_field.dart:2:8: Error: Dart library 'dart:js_interop' is not available on this platform.

When I try to build for mobile after importing the flutter_stripe_web library I get the interlop error. how can import the library but also build for mobile>

**Comment by jonasbark** [comment]
https://www.google.com/search?q=dart+conditional+import this should help you

**Comment by vuleanh** [comment]
Hi guys, is there any update on this?
